src/dmd/expressionsem.d

```diff
@@ -3448,7 +3448,8 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
                 return setError();
             }
 
-            if (!tf.callMatch(null, exp.arguments))
+            size_t failIndex;
```

Comment:
It's called `failIndex` here, but `failedIndex` further up. Please be consistent. Also, above the `failedIndex` is a pointer and here it is a value. Either name the one above `pFailedIndex' or make it a ref type. Probably the latter is better.

Reply:
If it's a `ref` parameter, what do I pass when I don't care about `failedIndex`?

Reply:
Pass a dummy variable.

Reply:
Then we get memory allocated for the unwanted error message whenever an overload fails to match, even though one overload does match, so no error message is needed.